libs/core/src/components/pds-input/pds-input.scss

```diff
@@ -97,13 +153,62 @@
 .pds-input__error-message,
 .pds-input__helper-message {
   font: var(--pine-typography-body-sm-medium);
-  margin-block-end: 0;
-  margin-block-start: var(--pine-dimension-2xs)
+  margin-block-end: var(--pine-dimension-none);
+  margin-block-start: var(--pine-dimension-2xs);
 }
 
 .pds-input__error-message {
   align-items: center;
-  color: var(--pine-color-text-message-danger);
+  color: var(--pds-input-error-color);
   display: flex;
   gap: var(--pine-dimension-2xs);
 }
+
+.pds-input__prefix,
+.pds-input__suffix {
```

Comment:
Should the prefix/suffix element styles change when disabled, readonly, invalid, etc?